enhancements/microshift/microshift-greenboot.md

```diff
@@ -0,0 +1,216 @@
+---
+title: MicroShift Greenboot
+authors:
+  - ggiguash
+reviewers:
```

Comment:
We should get someone from the RHEL team to look this over, too, in case they have advice for us.

Reply:
@runcom has been added to the reviewer list.